iosApp/iosApp/iOSApp.swift

```diff
@@ -4,9 +4,9 @@ import Shared
 @main
 struct iOSApp: App {
     init() {
-        KoinHelperKt.doInitKoin()
+        KoinHelper().doInitKoin()
```

Comment:
Why was this change made here? Won't this cause a problem with a new `KoinHelper` instance being created here which is then just thrown away?

Reply:
I'm unsure how it actually worked before! KoinHelper will only be instantiated once, when the iOSApp is initialized. @zsmb13